content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,443 @@ categories:
 
-#### FLAT
+| Parameter        | Description                                                                                                  | Options                     |
+|:-----------------|:-------------------------------------------------------------------------------------------------------------|:----------------------------|
+| `HYBRID_POLICY`  | Specifies the filter mode to use during vector search with filters (hybrid).                                 | `BATCHES`, `ADHOC_BF`       |
+| `BATCH_SIZE`     | A fixed batch size to use in every iteration when the `BATCHES` policy is auto-selected or requested.        | Positive integer.            |
 
-Currently, no runtime parameters are available for FLAT indexes.
 
-#### HNSW
+### Index-specific query parameters
 
-Optional parameters are:
+**FLAT**
 
-* `EF_RUNTIME` - The number of maximum top candidates to hold during the KNN search. Higher values of `EF_RUNTIME` will lead to a more accurate results on the expense of a longer runtime. Defaults to the `EF_RUNTIME` value passed on creation (which defaults to 10).
+Currently, there are no runtime params available for FLAT indexes.
 
-* `EPSILON` - Relative factor that sets the boundaries in which a range query may search for candidates. That is, vector candidates whose distance from the query vector is `radius*(1 + EPSILON)` are potentially scanned, allowing more extensive search and more accurate results (on the expense of runtime). Defaults to the `EPSILON` value passed on creation (which defaults to 0.01).
+**HNSW**
+
+Optional runtime params for HNSW indexes are:
+
+| Parameter       | Description                                                                                               | Default value       |
+|:----------------|:----------------------------------------------------------------------------------------------------------|:--------------------|
+| `EF_RUNTIME`    | The maximum number of top candidates to hold during the KNN search. Higher values lead to more accurate results at the expense of a longer query runtime. | Value passed during index creation (default is 10). |
+| `EPSILON`       | Relative factor that sets the boundaries for a vector range query. Vector candidates whose distance from the query vector is `radius * (1 + EPSILON)` are potentially scanned, allowing a more extensive search and more accurate results at the expense of runtime. | Value passed during index creation (default is 0.01). |
+
+
+
+### Important notes
 
 {{% alert title="Important notes" color="info" %}}
 
-1. Although specifying `K` requested results in KNN search, the default `LIMIT` is 10, to get all the returned results, specify `LIMIT 0 <K>` in your command.
+1. Although specifying `<top_k>` requested results in KNN search, the default `LIMIT` is 10, to get all the returned results, specify `LIMIT 0 <top_k>` in your search command. See examples below.
```

Comment:
I'm having a really hard time parsing this sentence. I suggest reading it aloud. This should probably be broken up into a 2-3 shorter sentences, with clearer subjects and predicates.

Reply:
I think this is just a typo with an unintentional comma where there was supposed to be a period.